packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-slug.js

```diff
@@ -41,9 +41,7 @@ export default function PageSlug( { postType, postId } ) {
 		( select ) => {
 			const { getEntityRecord, getEditedEntityRecord, getPostType } =
 				select( coreStore );
-			const savedRecord = getEntityRecord( 'postType', postType, postId, {
-				_fields: 'slug,generated_slug',
-			} );
```

Comment:
Using fields for one selector triggers two separate HTTP requests, and since we're already grabbing whole records, let's dedupe it.

## Screenshots
**Before**
![CleanShot 2023-11-02 at 14 43 57](https://github.com/WordPress/gutenberg/assets/240569/048c4be5-d54f-453d-b7d1-faf36667e220)

**After**
![CleanShot 2023-11-02 at 14 44 57](https://github.com/WordPress/gutenberg/assets/240569/11fa5b3d-9fa1-4c1a-8d44-7a9ce9df8032)